neo/tools/debugger/debugger.cpp

```diff
@@ -163,9 +167,16 @@ bool DebuggerServerInit ( void )
 		gDebuggerServer = NULL;
 		return false;
 	}
+	
```

Comment:
at the beginning of `DebuggerServerInit()`, please add an early return for `if ( gDebuggerServer != NULL )`, so we don't accidentally create rvDebuggerServer multiple times, e.g. if `com_enableDebuggerServer` thinks it has changed but hasn't (not sure if that actually happens, but better safe than sorry)

Reply:
im not sure what you mean, you did this part . 
AFAIK, there is already an check earlier that makes sure the debugger server isnt launched twice.

Reply:
there's a `if ( com_editors & EDITOR_DEBUGGER ) return false;` check, but I think that `com_editors & EDITOR_DEBUGGER` is only set if the debugger-server is actually connected to the debugger-client (at least that bit is cleared when disconnecting) - and obviously it can run without the client being connected